framework/pages/documentViewer/documentViewer.ts

```diff
@@ -0,0 +1,36 @@
+import {expect, type Page} from "@playwright/test";
+import {clickOnElement, fillInInput } from "../../helpers/utils.ts";
+
+const saveDocumentButton = 'save-document-button-352297';
+const saveDocumentButtonValue = '//*[@data-test-selector="save-document-button-8279"]';
```

Comment:
```suggestion
const saveDocumentButtonValue = '(//*[contains(@data-test-selector, "save-document-button-")])[1]';
```
the Id in selector isn't static, to make selector more universal